test/Microsoft.DotNet.ProjectModel.Loader.Tests/project.json

```diff
@@ -0,0 +1,29 @@
+﻿{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "keyFile": "../../tools/test_key.snk"
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc3-*"
```

Comment:
We shouldn't be using * versions anymore, now that NETCore.App is split from this repo.